src/test-helper.js

```diff
@@ -156,16 +157,19 @@ export async function updateTestUser(container, user) {
 export async function createTestRoom(container, roomValues = {}) {
   const db = container.get(Database);
 
+  const now = new Date();
+  const creatorAndOwner = uniqueId.create();
+
   const room = {
     _id: roomValues._id || uniqueId.create(),
     slug: roomValues.slug || '',
     name: roomValues.name || 'my-room',
     shortDescription: roomValues.shortDescription || '',
     isCollaborative: roomValues.isCollaborative || false,
-    ownedBy: roomValues.ownedBy || uniqueId.create(),
-    createdBy: roomValues.createdBy || uniqueId.create(),
-    createdOn: roomValues.createdOn || new Date(),
-    updatedOn: roomValues.updatedOn || new Date(),
+    ownedBy: roomValues.ownedBy || creatorAndOwner,
+    createdBy: roomValues.createdBy || creatorAndOwner,
```

Comment:
Yet another point in which we anticipated a direction that in the end never came.

Reply:
I had the same thought :-)